Source/macOS/LoopbackHTTPServer/OIDLoopbackHTTPServer.m

```diff
@@ -95,8 +93,7 @@ - (id)initWithPeerAddress:(NSData *)addr inputStream:(NSInputStream *)istr outpu
 
 - (void)dealloc {
     [self invalidate];
-    [peerAddress release];
-    [super dealloc];
+    peerAddress = nil;
```

Comment:
Optional nit (here and everywhere else) I think setting ivars to nil in a dealloc is not really needed.